src/main/java/pong/Player.java

```diff
@@ -0,0 +1,5 @@
+package pong;
+
+public enum Player {
+    HUMAN, COMPUTER
```

Comment:
would USER be better than HUMAN?

Reply:
I changed it to this in a more recent commit.

Reply:
Not sure what you mean. You updated it to user in a more recent commit?

Reply:
Sorry, you are right it is fixed now in commit [e3b7c13](https://github.com/lliebman/Pong/pull/4/commits/e3b7c130321d8c91261a3abb6ff1d5e20209f16d)